pkg/monitors/kubernetes/events/events.go

```diff
@@ -151,22 +152,27 @@ func (m *Monitor) shouldSendEvent(ev *v1.Event) bool {
 func (m *Monitor) handleNewEvent(ev *v1.Event) {
 	if m.shouldSendEvent(ev) {
 		sfxEvent := k8sEventToSignalFxEvent(ev)
+		sfxEvent.Properties[dpmeta.NotHostSpecificMeta] = true
 		m.Output.SendEvent(sfxEvent)
 	}
 }
 
 func k8sEventToSignalFxEvent(ev *v1.Event) *event.Event {
 	dims := map[string]string{
-		"kubernetes_kind":      ev.InvolvedObject.Kind,
+		"kubernetes_workload":  ev.InvolvedObject.Kind,
```

Comment:
Isn't this TitleCase whereas our agent's property is lowercase?

Reply:
This seems semantically questionable since our agent uses this property to describe the workload of a pod, whereas here it would be describing the type of the workload that the event pertains to.

Reply:
In the `kubernetes-cluster` monitor though we are talking about metrics, so we wanted the `workload` to be set to whatever object is running the pod. 
For events, they are always about the object themselves. StatefulSet's will have a "Created" event, which will create a pod and we will then get another "Created" event for a pod.

If we want events to have the same "workload" meaning as our metrics, we would need to extend the cluster cache to this monitor too. 
Maybe we could have an external "Cluster State" sort of singleton running in of the elected master node that could be used for these types of linking objects around issues.